client/plots/singleCellPlot.js

```diff
@@ -109,12 +109,13 @@ class singleCellPlot {
 
 		const q = state.termdbConfig.queries
 		this.opts.holder.style('position', 'relative')
-		this.insertBeforeId = `${this.id}-sandbox`
+		this.mainDivId = `${this.id}-sandbox`
 		const mainDiv = this.opts.holder
 			.insert('div')
 			.style('display', 'inline-block')
 			.style('vertical-align', 'top')
-			.attr('id', this.insertBeforeId)
+			.style('white-space', 'nowrap')
+			.attr('id', this.mainDivId)
```

Comment:
Like Congyu mentioned, we need to solve this. <img width="1246" alt="image013" src="https://github.com/user-attachments/assets/e1105d53-fed4-4d74-b5c3-12242135dfc7" />

Reply:
it will be in another PR